scripts/word_embeddings/evaluate_pretrained.py

```diff
@@ -198,14 +198,16 @@ def enforce_max_size(token_embedding, size):
 
     enforce_max_size(token_embedding_, args_.max_vocab_size)
     known_tokens = set(token_embedding_.idx_to_token)
-    # Auto-extend token_embedding with unknown extra eval tokens
+
+    # Extend token_embedding with unknown extra eval tokens
```

Comment:
Add a period.

Reply:
We should follow PEP 8 https://www.python.org/dev/peps/pep-0008/#inline-comments

Reply:
LGTM